pom.xml

```diff
@@ -381,7 +381,7 @@
         <version.org.jboss.ejb-client-legacy>3.0.3.Final</version.org.jboss.ejb-client-legacy>
         <version.org.jboss.ejb3.ext-api>2.3.0.Final</version.org.jboss.ejb3.ext-api>
         <version.org.jboss.genericjms>2.0.8.Final</version.org.jboss.genericjms>
-        <version.org.jboss.hal.console>3.2.11.Final</version.org.jboss.hal.console>
+        <version.org.jboss.hal.console>3.5.12.Final</version.org.jboss.hal.console>
```

Comment:
as the update is quite large, please run the build with the tests and see if it is ok